src/main/resources/react4xp/components/layouts/TwoColumnProcessor.ts

```diff
@@ -0,0 +1,24 @@
+import type { LayoutComponentProcessorFunction } from '@enonic-types/lib-react4xp/DataFetcher';
+
+export const layoutProcessor: LayoutComponentProcessorFunction<'com.enonic.app.hmdb:2-column'> = ({ content, component, }) => {
+    const region = component?.regions|| {};
+
+    const { regions } = component;
```

Comment:
Why do you need to define regions 2 times with the same value ? Calling one of them region is also misleading.